variables.tf

```diff
@@ -31,6 +31,16 @@ variable "vault_path" {
   default     = ""
 }
 
+variable "vault_login_path" {
```

Comment:
OK, so this is the varaiable referred to in the other PR, so now I see where it comes from, but why do we need it? What would a typical value for this be? The changes look reasonable I just want to make sure I understand what's going on here.

Reply:
@massdosage  this variable will allow passing the value of `-path` flag in `vault login` command.  This value can differ depends on vault setup in accounts.